Automattic/newspack-plugin
# Hide Campaigns plugin in menu; display settings in wizard

### All Submissions:






### Changes proposed in this Pull Request:



Hides the Campaigns plugin item from main menu, adds the Campaigns plugin settings view to the Campaigns wizard. 

### How to test the changes in this Pull Request:

1. Open WP Admin, verify the Campaigns plugin menu item is not present in the left menu
2. Open the Campaigns wizard, observe a new tab – "Settings"
3. Update the settings, verify they are updated in the (hidden in menu) Campaigns settings screen at `/wp-admin/edit.php?post_type=newspack_popups_cpt&page=newspack-popups-settings-admin` 

### Other information:
